source/Clean.js

```diff
@@ -30,7 +30,7 @@ var styleToSemantic = {
         }
     },
     fontWeight: {
-        regexp: /^bold/i,
+        regexp: /^bold|^700/i,
```

Comment:
this is interesting, that number is probably coming from GDocs, I remember a font weight of 700